packages/expo-video/android/src/main/java/expo/modules/video/playbackService/ExpoVideoPlaybackService.kt

```diff
@@ -130,33 +185,55 @@ class ExpoVideoPlaybackService : MediaSessionService() {
       .setStyle(MediaStyleNotificationHelper.MediaStyle(session))
       .build()
 
-    // Each of the players has it's own notification when playing.
-    notificationManager.notify(session.player.hashCode(), notificationCompat)
+    val notificationId = session.player.hashCode()
+
+    if (startInForegroundRequired) {
+      try {
+        startForeground(notificationId, notificationCompat)
+        isForeground = true
+      } catch (_: Exception) {
+        notificationManager.notify(notificationId, notificationCompat)
```

Comment:
Should we log or present that exception to the user? It is better to not show the notification if that service wasn't started as foreground service. Users might think that everything is okay even if something failed.